src/fable-fsharp/Replacements.fs

```diff
@@ -190,6 +193,8 @@ module Util =
                     |> makeCall com i.range (Fable.PrimitiveType (Fable.Number Int32))
                 Fable.Apply(op, [comp; makeConst 0], Fable.ApplyMeth, i.returnType, i.range)
                 |> Some
+            | Fable.Class None ->  // For enums only ("DeclaredType <enumName> Class null")
+                Fable.Apply(op, args, Fable.ApplyMeth, i.returnType, i.range) |> Some  
```

Comment:
Why is this necessary? Are some enum types escaping `Fable.Wrapped (Type (Fable.PrimitiveType _), _)`?